Feature/paypal refund

daniel-rose opened the pull request.

daniel-rose committed:
Update composer.json

daniel-rose committed:
use only lower letters for packages

daniel-rose committed:
start from zero

daniel-rose committed:
remove phpName and refPhpName from foreign-key

daniel-rose committed:
Merge remote-tracking branch 'origin/feature/paypal-refund' into feature/paypal-refund

# Conflicts:
#	composer.json
#	src/FondOfSpryker/Zed/Creditmemo/Persistence/Propel/Schema/fos_creditmemo.schema.xml

daniel-rose committed:
- remove unused file
- CreditMemo instead of Creditmemo

daniel-rose committed:
- CreditMemo instead of Creditmemo

daniel-rose committed:
- CreditMemo instead of Creditmemo

daniel-rose committed:
- use CreditMemo... instead of Creditmemo...
- add own reference to credit memo (pre plugin)
- add items and address via plugin (post & pre)
- use item and address transfer instead of own transfer models
- add credit-memo-extension as dependency
- cleanup dependencies
- add oms plugin
- use bridges for external facades

daniel-rose committed:
implements all method from interface

daniel-rose committed:
remove phpName

daniel-rose committed:
- use correct db and transfer fields
- generate correct reference

daniel-rose committed:
- add oms as required dependency
- use correct reference name value
- fix typo
- ...

julianhyatt commented on src/FondOfSpryker/Zed/CreditMemo/Business/CreditMemoBusinessFactory.php:
```diff
@@ -0,0 +1,108 @@
+    }
+
+    /**
+     * @return \FondOfSpryker\Zed\CreditMemoExtension\Dependency\Plugin\CreditMemoPreSavePluginInterface]|
```
gehört das dahin? ]|

daniel-rose committed:
fix typo in return tag

daniel-rose committed:
use const instead of fix string

Commit:
- credit memo preparations

Commit:
- Removed processor plugin

Commit:
- removed address from credit memo

Commit:
- add locale to credit memo

Commit:
- save more data

Commit:
- phpcs fixes
- added facade method to get sales order items by creditmemo

julianhyatt committed:
Merge pull request #2 from fond-of/feature/refund-facade-method-for-command

features

julianhyatt committed:
feature (#3)

* - more logging
- added limit and offset
- added config option to set max value for processing refunds at once
- refactoring

* - refactor
